.github/workflows/vercel-deployment.yml

```diff
@@ -8,15 +8,5 @@ on:
   push:
 
 jobs:
-  Deploy-Production:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Install Vercel CLI
-        run: npm install --global vercel@latest
-      - name: Pull Vercel Environment Information
-        run: vercel pull --yes --environment=production --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Build Project Artifacts
-        run: vercel build --prod --token=${{ secrets.VERCEL_TOKEN }}
-      - name: Deploy Project Artifacts to Vercel
-        run: vercel deploy --prebuilt --prod --token=${{ secrets.VERCEL_TOKEN }}
+  - name: Deploy to vercel
+    uses: ai-cfia/github-workflows/.github/workflows/workflow-vercel-deployment.yml@39-create-vercel-deployment-workflow
```

Comment:
expecting a parameter specifying the environment dev, uat or prd here?

dev should get pushed for any branch (we might want to have dev-<BRANCH NAME>)
uat based on main
prd not yet because we don't have a production environment yet.

Reply:
do you mean like a proposed in my comment? or a parameter to the command?